web/src/components/countrytable.js

```diff
@@ -329,10 +329,12 @@ const CountryElectricityProductionTable = React.memo(({
 
   // Use the whole history to determine the min/max,
   // fallback on current data
-  const history = useCurrentZoneHistory();
+  const history = useCurrentZoneHistory() || [];
```

Comment:
Required as `useCurrentZoneHistory` currently falls back to returning `[]`.
However, we don't want to have this function assume that it always fallsback to `[]`, so we added a redundant fullback here to always make sure it returns `[]` in case of missing data.